### ISSUE TITLE
Problems with asynchronous generators and iterators

### ISSUE THREAD

**Issue by benjaminkitt** [wallabyjs/quokka#165]
I am unable to get asynchronous generators and iterators to work with quokka using Node 10 nightly. This code executes properly when run via the node cli.

### Sample code

```javascript
'use strict'

const { get } = require('http')

function getTaco() {
  return new Promise((resolve, reject) => {
    let body = '';
    const req = get('http://taco-randomizer.herokuapp.com/random/', (res) => {
      res.on('data', data => body += data)
      res.on('error', err => reject(err))
      res.on('end', () => resolve(JSON.parse(body)))
    })
  })
}

async function* tacoFactory(howMany) {
  for (let i = 0; i < howMany; i++) {
    let taco
    try {
      taco = await getTaco()
      yield taco
    } catch (err) {
      console.error(err)
    }
  }
}

async function giveMeSomeTacos(howMany) {
  console.time('Tacos retrieved')
  for await (let taco of tacoFactory(howMany)) {
    const description = ['shell', 'base_layer', 'mixin', 'seasoning', 'condiment'].map(layer => taco[layer].name).join(', ')
    console.log(description)
  }
  console.timeEnd('Tacos retrieved')
}

giveMeSomeTacos(10)
```

### Quokka.js Console Output

```
​​​​Quokka #2 (node: v10.0.0-nightly20180223a29089d7c8)​​​​
​​​​
​​tacoFactory is not a function​​
  at ​​​giveMeSomeTacos​​​ ​01-async-iterators.js:33:25​
  at ​​​Object.<anonymous>​​​ ​01-async-iterators.js:40:0​
  at ​​​Object.<anonymous>​​​ ​01-async-iterators.js:1​
```

### Code editor version

Visual Studio Code v1.20.1

### OS name and version

OSX 10.12.6

**Comment by ArtemGovorov** [comment]
Thanks for providing the Quokka code. The issue is fixed and the fixed is published in Quokka extension v1.0.108.

**Comment by benjaminkitt** [comment]
Wow, thanks for the fast turnaround!
